server/utils/bcrypt.ts

```diff
@@ -0,0 +1,20 @@
+import { hash, compare, genSalt } from "bcryptjs";
+
+export const createHashPassword = async (password: string): Promise<string> => {
+  try {
```

Comment:
no need to use a try-catch block, an error will handled by the parent catch block automatically

Reply:
fixed